_limitations/system_function_limitations.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Unsupported Functions, Views, and Procedures 
+nav_order: 3
+---
+
+## Unsupported functions, views, and procedures
+
+Babelfish provides limited support for SQL Server system-implemented functions, views, and procedures. The following lists detail the functionality implemented for each version of Babelfish (and subsequent releases).
+
+### Built-in Functions
+
+The following Built-in Functions are supported for versions 2.1.0 and higher: 
+
+- IS_MEMBER()
+- IS_ROLEMEMBER()
+- HAS_PERMS_BY_NAME() 
+
+The following Built-in functions are supported for versions 1.2.0 and higher: 
+
+- COLUMNS_UPDATED()
+- UPDATE()
+- FULLTEXTSERVICEPROPERTY()
+- ISJSON()
+- JSON_QUERY()
+- JSON_VALUE()
+- HAS_DBACCESS()
+- SUSER_SID()
+- SUSER_SNAME()
+- IS_SRVROLEMEMBER()
+
+The following Built-in functions are supported for versions 1.0.0 and higher:
```

Comment:
this list is not complete. There are many more BIFs in 1.0.0 and also additional BIFs were added in 1.1.0

Reply:
I'd like this to eventually be a list of those BIFs we don't support (instead of supported), instead of BIFs we do, with the idea being we can help our users with workarounds for individual Fs as needed...